### DISCUSSION TITLE
Artillery expect plugin "Failed expectations for request" is vague

### DISCUSSION THREAD

**Discussion by jfstephe** [artilleryio/artillery#1537] [Artillery]
Given a yaml snippet as below:

```      - get:
          url: "/api/someEndpoint"
          headers:
            Content-Type: "application/json"
            Accept: application/json
          capture:
            - json: "$.data.displayName"
              as: "anElement"
            - json: "$.state"
              as: "state"

          expect:
            - contentType: "application/json; charset=utf-8"
            - equals:
              - "{{ anElement }}"
              - "Some Test Value"
            - equals:
              - "{{ state }}"
              - "1"
```

When the expectations aren't met you get an error entry along the lines of:

Failed expectations for request <URL>/api/someEndpoint":1}

Which isn't as helpful as it perhaps could be. It would be great if it could state which expected/actual values where compared that failed the test.

**Comment by hassy** [comment]
With the default `pretty` formatter, Artillery _should_ be printing full details of a failing expectation (expected vs got):

https://www.artillery.io/docs/guides/plugins/plugin-expectations-assertions#formatters

(how quickly do 6 months fly past!?)
